packages/ui-components/src/components/description-list/description-list.tsx

```diff
@@ -0,0 +1,38 @@
+import { Component, Fragment, Host, Prop, h } from '@stencil/core';
+import { IDescriptionList, IDescriptionListItem } from './description-list.types';
+import { TEXT_TOOLTIP_CONFIG } from './description-list.config';
+import { ETooltipPosition } from '../../types';
+
+@Component({
+	tag: 'kv-description-list',
+	styleUrl: 'description-list.scss',
+	shadow: true
+})
+export class KvDescriptionList implements IDescriptionList {
+	/** @inheritdoc */
+	@Prop({ reflect: true }) items!: IDescriptionListItem[];
+
+	render() {
+		return (
+			<Host>
+				<div class="description-list-container">
+					{this.items?.map(({ title, description, tooltipInfo }) => (
+						<Fragment>
+							<div class="title">{title}</div>
+							<div class="description">
+								<kv-tooltip text={tooltipInfo?.text && !tooltipInfo.icon ? tooltipInfo.text : ''} options={TEXT_TOOLTIP_CONFIG}>
```

Comment:
This seems very odd to me. Do we only want to show text if it exists and if there's no icon? 🤔

At least move this a util so it becomes clear what you want to do here.

Also, seems to me that this condition should apply to the entire `kv-tooltip`, not only the text prop.

Reply:
Also, seems to me that this tooltip config will need to be configurable via prop.

Reply:
> This seems very odd to me. Do we only want to show text if it exists and if there's no icon? 🤔
That is indeed the intended behaviour, if there's text but no icon provided, show it on the tooltip.

> At least move this a util so it becomes clear what you want to do here.
I will this verification to a helper/util method so it becomes clearer.

> Also, seems to me that this condition should apply to the entire `kv-tooltip`, not only the text prop.
My bad but I can't seem to understand what you mean by this, are we talking about conditionally rendering this tooltip?